### PR TITLE
Work

### PR DESCRIPTION
**Pull request by Mwapsam**
- Configure Linters (Webhint, stylelint, lighthouse & eslint)
- Add html, css & JavaScript file
- Fix linter errors
- Save to git pages
- Update Readme file

### CODE REVIEW

File: about.html [hunk 1]
```diff
@@ -0,0 +1,174 @@
...
+                alt="coincity Logo"
+              />
+            </a>
+          </div>
+          <div class="patents">
+            Copyright © 2021 All rights reserved | coincity
+            <h5 class="footerText">
+              Using blockchains to implement new and experimental forms of
+              ownership for land and other scarce assets, as well as new and
+              experimental forms of democratic governance.
```
Comment (leolpaz) [inline about.html]: - [ ] To comply with the design, kindly make sure that both footers have the same text content

File: assets/css/main.css [hunk 2]
```diff
@@ -0,0 +1,522 @@
...
+  .footerText {
+    font-family: 'Lato', sans-serif;
+    line-height: 1.2;
+    color: #a5a1a1;
+    font-size: 16px;
+  }
+
+  .aboutFooter {
+    background: #000;
+    height: 300px;
```
Comment (leolpaz) [inline assets/css/main.css]: - [ ] The footer on both pages should look the same, currently, the one on the about page is a lot taller than the one on the home page, kindly fix that
About page
image
Home page
image



- [ ] The footer on the mobile version in the about page should have a light background, kindly add that
image

File: assets/css/desktop.css [hunk 3]
```diff
@@ -0,0 +1,540 @@
...
+    color: #504f4f;
+    font-size: 20px;
+  }
+
+  .imageDiv h3 {
+    font-family: 'Lato', sans-serif;
+    color: #d84d40;
+    font-weight: 500;
+    font-size: 30px;
+  }
```
Comment (leolpaz) [inline assets/css/desktop.css]: - [x] Kindly fix the spacing between the h3 and the h1
Your page:
image
Design:
image

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @Mwapsam , 

You've done an amazing job so far, I'd like to highlight the following:
- Linters is passing
- Good video

Some issues need to be resolved before your approval, but you're almost there!



## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by julie-ify — APPROVED** [review]
## Hi @Mwapsam, 

# Highlights
- Good job implementing the required changes from the previous reviewer ✔️ 
- Good video presentation ✔️ 

# Status: Approved ✔️ 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved emoji

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**